### PR TITLE
feat: Support for selecting batch export model

### PR DESCRIPTION
**Pull request by tomasfarias**
## Problem

We want to support new models in batch exports, particularly the person model.

## Changes

This PR adds baseline support for a person model by adding a select input to both configuration forms and a field in the batch export model. The select input is behind a new feature flag, and the new field is not used.

Originally, I also intended to add custom schema support, but ran into conflicts with HogQL, so I scrapped that. First commit shows this work.

Old configuration form has a dropdown below name:
image

Didn't spend too much time making this pretty as maybe we will be on new UI by the time this is out.

For new UI we display a table with field information too: 
image

Would be nice to be able to customize the info.

👉 _Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review._

## Does this work well for both Cloud and self-hosted?
Yes

### CODE REVIEW

**Comment by tomasfarias** [comment]
@benjackwhite If these frontend changes conflict with yours, happy to get your stuff merged first and I can rebase.

**Review by benjackwhite — APPROVED** [review]
Looks good to me - I don't think this needs to wait for the other PR - the merge should be straightforward

File: frontend/src/queries/schema.json [hunk 1]
```diff
@@ -2400,6 +2400,29 @@
             "required": ["key", "operator", "type"],
             "type": "object"
         },
+        "DatabaseSchemaBatchExportTable": {
```
Comment (tomasfarias) [inline frontend/src/queries/schema.json:2403]: Didn't think I had to manually update this :thinking:

Comment (tomasfarias) [reply frontend/src/queries/schema.json:2403]: Annoyingly also have to update schema.py.

**Pull request merged**